spec/models/collection_spec.rb

```diff
@@ -1,4 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Collection, type: :model do
```

Comment:
Just a question, can we fix some other tests to use this new callback? like pocket's model tests

Reply:
It could be done in another story

Reply:
ok, please remind us in slack or add it as a story for this sprint